plan.md

```diff
@@ -0,0 +1,75 @@
+# Upgrade Procedure
```

Comment:
looks good. can you move this to proposal folder? Let's organize all the proposals info in one directory per proposal

Reply:
Something like: `/proposals/mainnet2-upgrade-proposal/upgrade-process.md`